pkgs/development/interpreters/ruby/default.nix

```diff
@@ -253,6 +260,7 @@ let
 in {
   ruby_2_7 = generic {
     version = rubyVersion "2" "7" "5" "";
+    revision = "f69aeb83146be640995753667fdd6c6f157527f5";
```

Comment:
Can you add a comment remembering people to also update this hash when they update the version? Otherwise I can see people forgetting to do this.

Reply:
or better we should have an update script. This will definitely be forgotten especially when using nix-update.